src/core/schema/src/main/scala/BinaryFileSchema.scala

```diff
@@ -1,29 +0,0 @@
-// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Please keep these in their original places since we'll be moving to Spark Images soon anyway.